### ISSUE TITLE
Home

### ISSUE THREAD

**Issue by mhuangdesign** [mhuangdesign/Angularjs#1]
<!doctype html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="make a form with Angularjs ">
    <meta name="keywords" content="Angular Agnularjs add remove search form ">

	<title>Home</title>
	<link rel="stylesheet" type="text/css" href="style.css" >
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
	<link href='https://fonts.googleapis.com/css?family=Merriweather:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
</head>
<body>	
	<div class="wrapper">
	<header>
		<h1>Test task!</h1>
		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elitsed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam</p>
	</header>
	<main>
		<h2>ADD AN ITEM</h2>
		<div class="columns" ng-app="myApp" ng-controller="myCtrl">
			
			<div class="col-left">
				
				<p><input type="text" id="enter" 
					placeholder="Enter item" 
					ng-model="enterItem"></p>
				
				 <select ng-model="selectColumn">
					    <option ng-repeat="column in columns" value="{{column}}">{{column}}</option>	
				</select>
				<br>
				
				<p><input type="button" id="add" value="Add item" ng-click="addName()" ></p>
				
				<p><label for="search">Search an item</label></p>
				<p><input type="search" id="search" name="search" placeholder="Search" ng-model="search"></p>
			</div>
			
			<div class="col-right">
				
				<div class="column-1">
					<h3>Column 1</h3>
					<ul>
						<li ng-repeat="itemLeft in Leftitems track by $index | filter:search">{{itemLeft}}
							<button type="button" ng-click="Leftitems.splice($index, 1)" aria-label="Remove">
								<i class="fa fa-times"></i></button></li>
					</ul> 
				</div>
				
				<div class="column-2">
					<h3>Column 2</h3>
					<ul>
						<li ng-repeat="itemRight in Rightitems track by $index | filter:search">{{itemRight}}
							<button type="button" ng-click="Rightitems.splice($index, 1)" aria-label="Remove">
								<i class="fa fa-times"></i></button></li>
					</ul>
				</div>
			</div>
	
		</div>
	</main>
	</div>
<script>
	var app = angular.module('myApp', []);
	app.controller('myCtrl', function($scope) {
		$scope.columns = ['column1','column2']
		$scope.Leftitems = ['Item','Item','Item','Item','Item'];
		$scope.Rightitems = ['Item','Item','Item'];
		$scope.addName = function(){
			if($scope.selectColumn == null || $scope.selectColumn == "" || $scope.enterItem == null || $scope.enterItem == ""){
				if(($scope.enterItem == null || $scope.enterItem == "") && ($scope.selectColumn == null || $scope.selectColumn == "")){
					alert("Please enter item name and select column!");
				}else if($scope.enterItem == null || $scope.enterItem == ""){ 
					alert("Please enter item name!"); 
				}else{ 
					alert("Please select column!");
				}
				
			}else if($scope.selectColumn=='column1' || $scope.selectColumn=='column2'){
				if($scope.selectColumn=='column1'){
					$scope.Leftitems.push($scope.enterItem);
				}
				if($scope.selectColumn=='column2'){
					$scope.Rightitems.push($scope.enterItem);
				}
			}else{alert('Error');}
			
		};
	});

</script>	
</body>

	
</html>
